docs/site/src/contributor.md

```diff
@@ -0,0 +1,20 @@
+# Contributor Manual
+
+Information about how to contribute code to the wallet.
+
+This includes:
+
+* Information about our **code** ("what"). This includes information about
+    
+    * coding style,
+    * libraries,
+    * tools,
+    
+    that we use, and which are not specific to our problem domain.
```

Comment:
Just curious: is this part meant to be read as a sentence?

If so, we should probably add an "and" after the penultimate item, and remove the comma after the final item:

```suggestion
    * coding style,
    * libraries, and
    * tools
    
    that we use, and which are not specific to our problem domain.
```

Otherwise it reads as "tools, that we use", which seems awkward.